web/src/translation/i18n.ts

```diff
@@ -25,7 +25,7 @@ i18n
 i18n.on('languageChanged', function (lng: keyof typeof localeToFacebookLocale) {
   document.documentElement.setAttribute('lang', lng);
   // TODO: Use react-helmet to manage meta tags
-  document.title = `Electricity Maps | ${i18n.t('misc.maintitle')}`;
+  document.title = `Electricity Maps | ${t('misc.maintitle')}`;
```

Comment:
Will this update when the language changes? I think that was the original reason we keept the i18n import here.

Reply:
It did in my testing but it would be great validate further